data/json/items/gun/50.json

```diff
@@ -31,7 +31,7 @@
     "copy-from": "rifle_auto",
     "looks_like": "modular_ar15",
     "type": "GUN",
-    "name": { "str_sp": "M2HB Browning HMG" },
+    "name": { "str": "M2HB Browning HMG" },
```

Comment:
HMG is more commonly understood than MGL, but I’d probably still call it a machine gun for sake of consistency. M2 HB machine gun, or M2 Browning HB machine gun.